src/proxyClient.js

```diff
@@ -129,13 +129,18 @@ var SUPPORT_BASE64_EMBEDDING;
 var filename;
 filename ||= '<<< filename >>>';
 
+#if SINGLE_FILE && SINGLE_FILE_BINARY_ENCODE
+#include "binaryDecode.js"
+var workerURL = URL.createObjectURL(new Blob([binaryDecode(filename)], {type: 'application/javascript'}));
+#else
 var workerURL = filename;
 if (SUPPORT_BASE64_EMBEDDING) {
```

Comment:
Unrelated to this change but this line looks very strange.  Shouldn't this be `#if SUPPORT_BASE64_EMBEDDING`?

Reply:
https://github.com/emscripten-core/emscripten/pull/22512